src/services/database.ts

```diff
@@ -93,3 +102,28 @@ const CriteriaIndexMap: Record<Exclude<SearchCriteria, SearchCriteria.ALL>, Tabl
   vin: 'VinIndex',
   trailerId: 'TrailerIdIndex',
 };
+export const postTechRecord = async (request: any) => {
+  if (process.env.AWS_SAM_LOCAL) {
+    return '123';
+  }
+  const command = {
+    TableName: tableName,
+    ConditionExpression: '#vin <> :vin AND #systemNumber <> :systemNumber',
+    ExpressionAttributeNames: {
+      '#vin': 'vin',
+      '#systemNumber': 'systemNumber',
+    },
+    ExpressionAttributeValues: {
+      ':vin': { S: request.vin },
+      ':systemNumber': { S: request.systemNumber },
+    },
+    Item: marshall(request as Record<string, AttributeValue>),
+  };
+  try {
```

Comment:
I think we can now strip this try catch, I placed this in to help debug but I *think* it will now work on the outer parent one. Means we can remove the return null too

Reply:
Keep the error logger as well? Or would ya like me to remove that?

Reply:
Yeah keep the log, just log and throw like the above 2 methods please.